libs/npm/core/src/components/table/interfaces/TableInternal.ts

```diff
@@ -68,11 +68,13 @@ export class TableInternal<T extends object> implements ITableInternal<T> {
 
     this.rows = sortRows(rows, this.sortOrder);
     if (this.groupBy) {
-      this.groups = groupBy(this.rows, (row) => (row.original as any)['value']).map((group) => ({
```

Comment:
'value' was hardcode here so grouping was not previously working; however, works now by passing a string of the key name to the groupBy property. Let me know if this is ok

Reply:
I think the hardcoded `value` is incorrect and due to my test data having the property.  If you instead use `this.groupBy(row.original)` what happens?

Reply:
Hmm seems to be having a few issues, will try to sort it